code/modules/reagents/machinery/centrifuge.dm

```diff
@@ -67,7 +67,7 @@
 	while(cans.len>0 && beaker.reagents.reagent_list.len>0)
 		var/obj/item/weapon/reagent_containers/C = cans[1]
 		var/datum/reagent/R = beaker.reagents.reagent_list[1]
-		beaker.reagents.trans_id_to(C,R.id,50)
+		beaker.reagents.trans_id_to(C,R.id,300)
```

Comment:
why not just beaker max volume?

Reply:
Jury rigged. Is it:
>
>beaker.reagents.trans_id_to(C,R.id,R.amount)
?

Reply:
`beaker.reagents.trans_id_to(C,R.id,R.volume)` I be lieve